spacelift/resource_context_test.go

```diff
@@ -79,6 +79,8 @@ func TestContextResource(t *testing.T) {
 				Config: config("new description"),
 				Check: Resource(
 					resourceName,
+					Attribute("id", Equals("provider-test-context-old-description")),   // ID shouldn't change on rename
+					Attribute("name", Equals("Provider test context new description")), // but the name should
```

Comment:
This method to test that the context didn't get recreated is a bit... _indirect_, but I don't _think_ there's a way to directly test that a resource hasn't been recreated using the plugin SDKv2; I think the new plugin Framework has this feature. But given Spacelift's behavior of the ID slug not changing when something is renamed, it's not a _terrible_ way to verify that.

Reply:
this is fine to me

Reply:
Yeah also fine by me 👍🏻